PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -12,12 +12,20 @@
   , 'inclusiveSecondaryVertexFinderFilteredNegativeTagInfos'
     # new candidate-based framework (supported with RECO/AOD/MiniAOD)
   , 'pfImpactParameterTagInfos'
+  , 'pfImpactParameterTagInfosAK8'
+  , 'pfImpactParameterTagInfosCA15'
   , 'pfSecondaryVertexTagInfos'
   , 'pfSecondaryVertexNegativeTagInfos'
   , 'pfInclusiveSecondaryVertexFinderTagInfos'
+  , 'pfInclusiveSecondaryVertexFinderTagInfosAK8'
```

Comment:
well, it should be clear that the cone around 0.3-0.5 has nothing to do
with QCD metrics motivating the CA15, AK8 or AK4.
The cone in b-tagging has to do with decay cone size of a reasonable-pt B
hadron, its lorentz factor and its mass. So opening the cone is not
necessarly the right thing to do.
On the other hand the fatjet tagging with a DR of 1.5 (assuming you want to
catch to B in it) has several failing assumptions in btag: the jetaxis has
nothing to do with B-Hadron direction, life-time sign is flipping a lot,
AVR reconstruction doesn't work (no big deal as default is IVF).
So in pratice:
- for fatjets that are coming from a single parton => you better keep
0.3-0.5
- for fatjets that contains two B, the b-tag of the fatjet is running,
perhaps even giving reasonable numbers but is just failing a lot of
assumptions used in the code (I'd say pointless to use)


On Tue, Jun 23, 2015 at 7:44 PM, Dinko Ferencek <notifications@github.com>
wrote:

> In PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py
> <https://github.com/vhbb/cmssw/pull/60#discussion_r33069574>:
>
> >    , 'pfSecondaryVertexTagInfos'
> >    , 'pfSecondaryVertexNegativeTagInfos'
> >    , 'pfInclusiveSecondaryVertexFinderTagInfos'
> > +  , 'pfInclusiveSecondaryVertexFinderTagInfosAK8'
>
> I hope the answer to why we would want to enlarge the dR cone is obvious.
> We are dealing with fat jets whose subjets or SV can be separated by more
> than the typical dR requirements used in the b-tagging code.
>
> Also note that PAT did not and still does not modify any of the b-tagging
> default cuts several of which are of the dR type. Here we have an example
> of something that was specifically trained for AK8 and CA15 jets and
> requires "non-standard" cuts so that's why the corresponding TagInfos have
> the jet identifiers in their names. However, as I mentioned when I sent you
> the first draft of these taggers, none of the names are set in stone and
> can be modified. In principle, yes, maybe we can have R8 and R15 instead of
> AK8 and CA15. None if this is in the standard Reco so we can update this 10
> times over. However, I think it is better to have a fixed set of reference
> config files then to allow PAT to modify everything on the fly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vhbb/cmssw/pull/60/files#r33069574>.
>